.github/workflows/unit_tests.yml

```
@@ -0,0 +1,66 @@
+        run: |
+          sudo apt-get update
+      - name: Install ovos dependencies
+        run: |
+          pip install ovos-plugin-manager
+      - name: Run unittests
+        run: |
+          pytest --cov=ovos-skill-speedtest --cov-report xml test
+      - name: Upload coverage
+        env:
+          CODECOV_TOKEN: ${{secrets.CODECOV_TOKEN}}
+        uses: codecov/codecov-action@v2
```

actionlint

the runner of "codecov/codecov-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)